<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCloudSetup</class>
 <widget class="QDialog" name="DialogCloudSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S3 Access ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S3 Secret Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S3 Bucket</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditS3Access">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditS3Secret">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditS4Bucket">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEMailUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEMailPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEMailServer">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail Server</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail Port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxEMailPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEMailFrom">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EMail Domain</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEMailDomain">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transport</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTransport">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="Plain Text">TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="Secure Sockets Layer">SSL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="Transport Layer Security">TLS</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditS3Access</tabstop>
  <tabstop>lineEditS3Secret</tabstop>
  <tabstop>lineEditS4Bucket</tabstop>
  <tabstop>lineEditEMailFrom</tabstop>
  <tabstop>lineEditEMailDomain</tabstop>
  <tabstop>lineEditEMailUser</tabstop>
  <tabstop>lineEditEMailPassword</tabstop>
  <tabstop>lineEditEMailServer</tabstop>
  <tabstop>spinBoxEMailPort</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCloudSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCloudSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
